docs/posts/_metadata.yml

```diff
@@ -11,6 +11,5 @@ title-block-banner: true
 comments:
   giscus:
     repo: ibis-project/ibis
-    theme: light
```

Comment:
Removing the explicit `light` theme makes the theme automatic based on whether the site theme is light or dark 🚀